docs/examples/test_examples.py

```diff
@@ -0,0 +1,268 @@
+            assert service_rpc.method() == None
+
+
+class TestFibonacci(object):
+
+    def test_fibonaccci(self, container_factory, rabbit_config):
+
+        from fibonacci import Fibonacci
+
+        container = container_factory(Fibonacci, rabbit_config)
+        container.start()
+
+        with ServiceRpcProxy('fibonacci', rabbit_config) as fibonacci_rpc:
+            for number in [1, 2, 3, 5, 8, 13]:
+                assert fibonacci_rpc.next() == number
+
+
+class TestHelloWorld(object):
+
+    def test_hello_world(self, container_factory, rabbit_config):
+
+        from helloworld import GreetingService
+
+        container = container_factory(GreetingService, rabbit_config)
+        container.start()
+
+        with ServiceRpcProxy('greeting_service', rabbit_config) as greet_rpc:
+            assert greet_rpc.hello("Matt") == "Hello, Matt!"
+
+
+class TestRpc(object):
+
+    def test_rpc(self, container_factory, rabbit_config):
+
+        from rpc import ServiceX, ServiceY
+
+        container_x = container_factory(ServiceX, rabbit_config)
+        container_y = container_factory(ServiceY, rabbit_config)
+        container_x.start()
+        container_y.start()
+
+        with ServiceRpcProxy('service_x', rabbit_config) as service_x_rpc:
+            assert service_x_rpc.remote_method("foo") == "foo-x-y"
+
+    def test_standalone_rpc(self, container_factory, rabbit_config):
+
+        from rpc import ServiceX, ServiceY
+
+        container_x = container_factory(ServiceX, rabbit_config)
+        container_y = container_factory(ServiceY, rabbit_config)
+        container_x.start()
+        container_y.start()
+
+        # standalone example doesn't import due to undefined variables
+        # use execfile with a local namespace
+        ns = {'AMQP_URI': rabbit_config['AMQP_URI']}
+
+        dirpath = os.path.dirname(os.path.realpath(__file__))
+        filepath = os.path.join(dirpath, 'standalone_rpc.py')
+
+        with entrypoint_waiter(container_x, 'remote_method'):
+            execfile(filepath, globals(), ns)
+
+    def test_async_rpc(self, container_factory, rabbit_config):
+
+        from rpc import ServiceX, ServiceY
+
+        container_x = container_factory(ServiceX, rabbit_config)
+        container_y = container_factory(ServiceY, rabbit_config)
+        container_x.start()
+        container_y.start()
+
+        # async example doesn't import due to undefined variables
+        # use execfile with a local namespace
+        from nameko.standalone.rpc import ClusterRpcProxy
+        ns = {
+            'config': rabbit_config,
+            'ClusterRpcProxy': ClusterRpcProxy
+        }
+
+        dirpath = os.path.dirname(os.path.realpath(__file__))
+        filepath = os.path.join(dirpath, 'async_rpc.py')
+
+        with entrypoint_waiter(container_x, 'remote_method'):
+            execfile(filepath, globals(), ns)
+
+
+class TestServiceContainer(object):
```

Comment:
They're present in the docs (http://nameko.readthedocs.org/en/latest/key_concepts.html#running-services) so I think they should be tested here.